docs/resources/access_control_rule_set.md

```diff
@@ -138,10 +138,138 @@ resource "databricks_access_control_rule_set" "automation_sp_rule_set" {
+
+  grant_rules {
+    principals = [data.databricks_user.john.acl_principal_id]
+    role       = "roles/group.manager"
+  }
+}
+```
+
+Through AWS Databricks account:
+
+```hcl
+locals {
+  account_id = "00000000-0000-0000-0000-000000000000"
+}
+
+// initialize provider at account-level
+provider "databricks" {
+  host       = "https://accounts.cloud.databricks.com"
+  account_id = local.account_id
+  username   = var.databricks_account_username
+  password   = var.databricks_account_password
+}
+
+// account level group creation
+resource "databricks_group" "ds" {
+  display_name = "Data Science"
+}
+
+data "databricks_user" "john" {
+  user_name = "john.doe@example.com"
+}
+
+resource "databricks_access_control_rule_set" "ds_group_rule_set" {
+  name = "accounts/${local.account_id}/groups/${databricks_group.ds.id}/ruleSets/default"
+
+  grant_rules {
+    principals = [data.databricks_user.john.acl_principal_id]
+    role       = "roles/group.manager"
+  }
+}
+```
+
+Through Azure Databricks account:
+
+```hcl
+locals {
+  account_id = "00000000-0000-0000-0000-000000000000"
+}
+
+// initialize provider at Azure account-level
+provider "databricks" {
+  host       = "https://accounts.azuredatabricks.net"
+  account_id = local.account_id
+  auth_type  = "azure-cli"
+}
+
+// account level group creation
+resource "databricks_group" "ds" {
+  display_name = "Data Science"
+}
+
+data "databricks_user" "john" {
+  user_name = "john.doe@example.com"
+}
+
+resource "databricks_access_control_rule_set" "ds_group_rule_set" {
+  name = "accounts/${local.account_id}/groups/${databricks_group.ds.id}/ruleSets/default"
+
+  grant_rules {
+    principals = [data.databricks_user.john.acl_principal_id]
+    role       = "roles/group.manager"
+  }
+}
+```
+
+Through GCP Databricks account:
+
+```hcl
+locals {
+  account_id = "00000000-0000-0000-0000-000000000000"
+}
+
+// initialize provider at account-level
+provider "databricks" {
+  host       = "https://accounts.gcp.databricks.com"
+  account_id = local.account_id
+}
+
+// account level group creation
+resource "databricks_group" "ds" {
+  display_name = "Data Science"
+}
+
+data "databricks_user" "john" {
+  user_name = "john.doe@example.com"
+}
+
+resource "databricks_access_control_rule_set" "ds_group_rule_set" {
+  name = "accounts/${local.account_id}/groups/${databricks_group.ds.id}/ruleSets/default"
+
+  grant_rules {
+    principals = [data.databricks_user.john.acl_principal_id]
+    role       = "roles/group.manager"
+  }
+}
+```
+
 ## Argument Reference
 
 * `name` - (Required) Unique identifier of a rule set. The name determines the resource to which the rule set applies. Currently, only default rule sets are supported. The following rule set formats are supported:
   * `accounts/{account_id}/servicePrincipals/{service_principal_application_id}/ruleSets/default`
+  * `accounts/{account_id}/groups/{group_id}/ruleSets/default`
```

Comment:
`{group_id}` is it ID or group name?

Reply:
the rule set for groups takes a group_id